src/components/labelled-date-time-input/LabelledDateTimeInput.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import DateTime from 'react-datetime'
+import moment from 'moment'
+import 'moment/locale/ru'
+import './LabelledDateTimeInput.css'
+
+const LabelledDateTimeInput = ({ label, value, timeFormat }) => {
+  return (
+    <label>
+      {label}{' '}
+      <DateTime timeFormat={timeFormat} defaultValue={value} locale="ru" />
+    </label>
+  )
+}
+
+LabelledDateTimeInput.propTypes = {
+  label: PropTypes.string.isRequired,
+  value: PropTypes.oneOfType([PropTypes.instanceOf(Date), PropTypes.object]),
```

Comment:
I think this prop should be named as `defaultValue`

Reply:
And let's replace `PropTypes.object` with `PropTypes.instanceOf(Moment)`. View this example:
![image](https://user-images.githubusercontent.com/25607790/63466589-3cf96480-c46c-11e9-91e0-fd21089dd370.png)

Reply:
done